public/data/chimay29hub.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Chinmay Sonawane",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/chinmay29hub.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/chinmay29hub",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/chinmay29hub",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/chinmay-sonawane-38474019a/",
```

```suggestion
      "url": "https://www.linkedin.com/in/chinmay-sonawane-38474019a",
```

The / is not needed in the url, it works anyway. It just gets added when you copy the url from the taskbar.

---

There's no harm in keeping the `/` here. I suggest we keep it as-is.